.travis.yml

```diff
@@ -1,12 +1,30 @@
 language: php
+
+sudo: false
+
 php:
   - 5.5
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
 env:
   global:
     secure: Yc+Xohkr/iEUU7FCQuSLXAE9ywNW9g6CfrM1Ki0Hl+fS15F3AXT7dFY8EyCJ4dP1/oI0dBmwrGWrltXV0XWIjGV1Ms3tefCgQpBBAqwT+hImzVP3RbpZW8Iyo2d0VgiDemQF1LWYD/pKu6d8WljTnv5D77NIMdEJjQ0uzeTLWdw=
-install: composer install --dev
+
+before_install:
+  - composer self-update
```

Comment:
@jpignata 

Composer generally emits warnings when it hasn't been updated in a while, and you may run into issues when you're using a different version of composer in development than what lives on Travis.

For example, when `composer` introduced the `^` operator for specifying version requirements, and locally, you started using that operator, but Travis wouldn't have that version yet, builds failed.

You can avoid this by either

* updating `composer` itself in the `before_install` section
* checking in `composer.phar` (see https://twitter.com/localheinz/status/581044053233524736)